express-joi-validation.js

```diff
@@ -72,9 +72,9 @@ module.exports.createValidator = function generateJoiMiddlewareInstance(cfg) {
 
     instance[type] = function(schema, opts) {
       opts = opts || {} // like config, default to empty object
-
+      const computedOpts = { ...container.joi, ...cfg.joi, ...opts.joi }
```

Comment:
👍  for this. The only change I would like is to remove the global `cfg.joi` since it's a little to prone to mistakes IMO. Best to have developers explicitly pass overrides in `opts.joi`

Reply:
Sorry I did not completely understand what you mean.
The `cfg.joi ` contains a global, pre-defined, set of default options for the underlying Joi module.

Reply:
Yep I understand. My concern is this might be confusing for developers. This module applies different defaults for query, body, etc as documented [here](https://github.com/evanshortiss/express-joi-validation#joi-options)

If a developer overrides those via global `cfg.joi` it could cause errors more easily IMO.

Though I see you still inherit from the `container.joi` so I think my initial concern is not so valid.

Let's keep it the way you have it 👍